### ISSUE TITLE
Copying wrong wallet address 

### ISSUE THREAD

**Issue by Varmab26** [Berry-Pool/nami-wallet#462]
I copied my wallet address but it was copying wrong address every time this leads to lost my 100ada

Uploading 20220316_000929.mp4…

**Comment by Varmab26** [comment]
https://user-images.githubusercontent.com/101715624/158585163-dc22e69e-ca50-46e8-a68c-cf6d5004299e.mp4

**Comment by kk-kien** [comment]
probably your computer got viruses

**Comment by alessandrokonrad** [comment]
Yes very likely a virus... this happened to a few others already.

**Comment by Varmab26** [comment]
But it copy correctly for different wallets
